apps/etterlatte-vilkaarsvurdering/src/main/kotlin/vilkaarsvurdering/migrering/MigreringRoutes.kt

```diff
@@ -0,0 +1,31 @@
+package no.nav.etterlatte.vilkaarsvurdering.migrering
+
+import io.ktor.http.HttpStatusCode
+import io.ktor.server.application.call
+import io.ktor.server.application.log
+import io.ktor.server.response.respond
+import io.ktor.server.routing.Route
+import io.ktor.server.routing.application
+import io.ktor.server.routing.patch
+import io.ktor.server.routing.route
+import no.nav.etterlatte.libs.common.BEHANDLINGSID_CALL_PARAMETER
+import no.nav.etterlatte.libs.common.vilkaarsvurdering.Utfall
+import no.nav.etterlatte.libs.common.withBehandlingId
+import no.nav.etterlatte.vilkaarsvurdering.klienter.BehandlingKlient
+
+fun Route.migrering(migreringService: MigreringService, behandlingKlient: BehandlingKlient) {
+    route("/api/vilkaarsvurdering/migrering") {
+        val logger = application.log
+
+        patch("/{$BEHANDLINGSID_CALL_PARAMETER}/vilkaar/utfall/{utfall}") {
```

Comment:
En tanke som slår meg her: hvis vi først skal ha en separat route for migrering, bør vi ikke isåfall gjøre alt her med en gang? Dvs, et kall som oppretter, setter utfall for vilkår + utfall? Høres ut som det burde forenkle det litt + minimere sannsynligheten for feil?

Reply:
Ja, sant. Lagde dette endepunktet ganske generisk no, så i prinsippet kan vi bruke det til kva som helst, men i praksis treng vi det jo berre for migrering.

Reply:
Kan gjera eit forsøk på å skrive om denne så alt går rett frå r&r-appen og hit, og at denne blir veldig spesialisert. Vil for så vidt gjera det enkelt for oss å slette unna igjen etterpå også.